src/Umbraco.Core/RuntimeOptions.cs

```diff
@@ -46,14 +46,12 @@ public static bool InstallMissingDatabase
         /// </summary>
         /// <remarks>
         /// <para>By default, when a database connection string is configured and it is possible to connect to
-        /// the database, but the database is empty, the runtime enters the BootFailed level. If this options
-        /// is set to true, it enters the Install level instead.</para>
-        /// <para>It is then up to the implementor, that is setting this value, to take over the installation
-        /// sequence.</para>
+        /// the database, but the database is empty, the runtime enters the Install level. If this options
+        /// is set to <c>false</c>, it enters the BootFailed level instead.</para>
         /// </remarks>
         public static bool InstallEmptyDatabase
         {
-            get => _installMissingDatabase ?? BoolSetting("Umbraco.Core.RuntimeState.InstallEmptyDatabase", false);
+            get => _installMissingDatabase ?? BoolSetting("Umbraco.Core.RuntimeState.InstallEmptyDatabase", true);
```

Comment:
I guess this is a 'breaking' change but is just something we need to document to say that this is true by default now. We can add the breaking change tag to this PR

Reply:
I guess it is, however you'll still need to have the version number in the config, configured the connection string and an empty database for the unattended install to kick in.